Makefile.authx

```diff
@@ -51,6 +74,43 @@ init-authx: mkdir
 	source ${PWD}/lib/keycloak/keycloak_setup.sh; \
 	export SERVICE=tyk; \
 	${PWD}/lib/tyk/tyk_setup.sh; \
-	echo  ; \
-	$(MAKE) compose-tyk \
-	${PWD}/lib/tyk/tyk_key_generation.sh;
+	echo ; \
+	$(MAKE) compose-tyk; \
+	source ${PWD}/lib/tyk/tyk_key_generation.sh; \
+	echo ;
+
+#>>>
+# authx, redeploy tyk
+
+#<<<
+redeploy-tyk: mkdir
```

Comment:
Redeploy should be alot cleaner than this. Is there no way to simply reconfigure a running keycloak and tyk instance?

Reply:
Keycloak is not redeployed.  

There likely is, definitely for Tyk as there are Tyk API endpoints for this.  I however found it easier to add a file and redeploy Tyk for now.